app/Http/Controllers/Forum/ForumThreadController.php

```
@@ -0,0 +1,218 @@
+use App\Services\SpamDetectionService;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Stevebauman\Purify\Facades\Purify;
+
+class ForumThreadController extends Controller
+{
+    /**
+     * Get a specific thread with its posts.
+     */
+    public function show(string $forumSlug, string $threadSlug): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$forumSlug'. (undefined)

(UnusedFormalParameter)